### ISSUE TITLE
bin2hex() expects parameter 1 to be string, resource given 

### ISSUE THREAD

**Issue by ctilluma** [Icinga/icingaweb2-module-director#1026]
When attempting to deploy a rendered config, an error message appears regarding bin2hex()

## Expected Behavior
Should deploy the rendered config with no errors.

## Current Behavior
Gives this error message regarding Bin2Hex()

## Steps to Reproduce (for bugs)
1. Fresh install of icinga2, icingaweb2, director
2. Config zones, api's, and run kickstart
3. Attempt to render and then deploy. Happens with a clean render or a render of objects ready to reploy.

## Your Environment

* Director version (System - About): 1.3.1
* Icinga Web 2 version and modules (System - About): 2.4.1
* Icinga 2 version (`icinga2 --version`): r2.6.3-1)
* Operating System and version: Debian 9 (Linux 4.9.0-3-amd64 #1 SMP Debian 4.9.30-2+deb9u2 (2017-06-26) x86_64 GNU/Linux)
* Webserver, PHP versions: PHP 7.0.19-1 (cli), Server version: Apache/2.4.25 (Debian)

**Comment by Thomas-Gelf** [comment]
@ctilluma: has a stack trace been shown? Did you render a completely empty config, or did you create some objects with Director? Are you using MySQL, MariaDB or PostgreSQL?

**Comment by ctilluma** [comment]
Thought I included the trace, but I don't see it above. This is with a clean install with no objects created. If I create objects, I still get the same issues.  

Here is the trace it provides:

bin2hex() expects parameter 1 to be string, resource given (DeploymentLogTable.php:28)

#0 [internal function]: Icinga\Application\ApplicationBootstrap->Icinga\Application\{closure}(2, 'bin2hex() expec...', '/usr/share/icin...', 28, Array)
#1 /usr/share/icingaweb2/modules/director/library/Director/Web/Table/DeploymentLogTable.php(28): bin2hex(Resource id #13)
#2 /usr/share/icingaweb2/modules/director/library/vendor/ipl/Web/Table/ZfQueryBasedTable.php(124): Icinga\Module\Director\Web\Table\DeploymentLogTable->renderRow(Object(stdClass))
#3 /usr/share/icingaweb2/modules/director/library/vendor/ipl/Web/Table/ZfQueryBasedTable.php(110): ipl\Web\Table\ZfQueryBasedTable->fetchRows()
#4 /usr/share/icingaweb2/modules/director/library/vendor/ipl/Html/BaseElement.php(133): ipl\Web\Table\ZfQueryBasedTable->renderContent()
#5 /usr/share/icingaweb2/modules/director/library/vendor/ipl/Html/Html.php(139): ipl\Html\BaseElement->render()
#6 /usr/share/icingaweb2/modules/director/library/vendor/ipl/Html/BaseElement.php(105): ipl\Html\Html->render()
#7 /usr/share/icingaweb2/modules/director/library/vendor/ipl/Html/BaseElement.php(133): ipl\Html\BaseElement->renderContent()
#8 /usr/share/icingaweb2/modules/director/library/vendor/ipl/Html/Html.php(227): ipl\Html\BaseElement->render()
#9 /usr/share/icingaweb2/modules/director/library/vendor/ipl/Zf1/SimpleViewRenderer.php(47): ipl\Html\Html->__toString()
#10 /usr/share/icingaweb2/modules/director/library/vendor/ipl/Zf1/SimpleViewRenderer.php(66): ipl\Zf1\SimpleViewRenderer->render()
#11 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action/HelperBroker.php(272): ipl\Zf1\SimpleViewRenderer->postDispatch()
#12 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action.php(518): Zend_Controller_Action_HelperBroker->notifyPostDispatch()
#13 /usr/share/php/Icinga/Web/Controller/Dispatcher.php(76): Zend_Controller_Action->dispatch('deploymentsActi...')
#14 /usr/share/icingaweb2/library/vendor/Zend/Controller/Front.php(937): Icinga\Web\Controller\Dispatcher->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#15 /usr/share/php/Icinga/Application/Web.php(389): Zend_Controller_Front->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#16 /usr/share/php/Icinga/Application/webrouter.php(109): Icinga\Application\Web->dispatch()
#17 /usr/share/icingaweb2/public/index.php(4): require_once('/usr/share/php/...')
#18 {main}

PostgreSQL 9.6

**Comment by ctilluma** [comment]
The table has 14 entries, so it at least was populated. It seems to have occurred from the initial kickstart. It's possible I got something wrong in the zone or api file during config. I'm gonna grab a couple hours sleep, been staring at other code too long. I can help debug if needed when I get up. It's a fresh install, so I don't mind going through the process again, but it's done this each time I've gone through an install, and I've tried a few different options each time from a setup perspective, making sure I purge the install, rm all directories, and drop the db each time. The other components are all packages, just director is a git clone.
